routers/web/org/projects.go

```diff
@@ -315,7 +315,7 @@ func EditProjectPost(ctx *context.Context) {
 	}
 }
 
-// ViewProject renders the project board for a project
+// ViewProject renders the project with board view for a project
```

Comment:
```suggestion
// ViewProject renders the project with column view for a project
```

Reply:
This is board view, not column view. Board View is a mode and we will have a Table View.